### ISSUE TITLE
Error message when booting with UEFI

### ISSUE THREAD

**Issue by reet-** [roburio/mugenhwcfg-live#2]
@mato Great project, very useful indeed!

I can confirm that the ISO dumped to an USB stick works for legacy BIOS boot (tested on Lenovo T480s and X260).

With UEFI, the following error is displayed:

![uefi-error](https://user-images.githubusercontent.com/5601828/76087014-99aa1000-5fb5-11ea-9421-fa6f91dadca2.png)

Reproducible via:
```
qemu-system-x86_64 -m 2048 -bios OVMF.fd -cdrom live-image-amd64.hybrid.iso -enable-kvm
```

Pressing enter twice boots the system.

**Comment by mato** [comment]
Indeed. This is due to UEFI boot using GRUB instead of ISOLINUX, and I've not done any customizations for the former. I've now figured out how to get QEMU+OVMF working and have reproduced this, so will look into it.

**Comment by mato** [comment]
Fixed in e3ce19f53d008d9e3d77678986e9b6f38c3024cb, I think. Please confirm on real hardware.

**Comment by reet-** [comment]
Works, thanks!
